### ISSUE TITLE
`arr.len` for high dimension arrays is wrong

### ISSUE THREAD

**Issue by SleepyRoy** [vlang/v#8089]
**V version:** 
master

**OS:** 
win10

**What did you do?**
```V
fn main() {
	arr := [[1,2,3], [4,5,6]]
	println(arr.len)
}
```

**What did you expect to see?**
[2, 3]
 
**What did you see instead?**
2

**Comment by SleepyRoy** [comment]
Obviously now `len` only records the first dimension of array.

**Comment by ntrel** [comment]
arr[0].len gives 3. I think this issue is invalid.

**Comment by SleepyRoy** [comment]
@ntrel yep I understand. I was just thinking about whether `len` of high dim arrays should record all dimensions instead of only first one...
